README.md

```diff
@@ -108,3 +108,46 @@
 	```
 	dfx --version
 	```
+
+## Workshop Proper
+
+1. Create a new project using the `dfx create` command in your terminal:
+
+```
+dfx new my_dapp
+cd my_dapp
+```
+This will create a new project directory named my_dapp and initialize it with a basic set of files and directories.
+
+2. Navigate to `main.mo` file inside `src/my_dapp_backend/` and you will see the generated `main.mo` file. This will be the entry point for your DApp.
+	```
+	actor {
+		public query func greet(name : Text) : async Text {
+			return "Hello, " # name # "!";
+		}; 
+	}; 
+	```
+This code defines a simple `greet` function that returns a greeting.
+
+3. Start the execution environment by running this command in terminal A.
+	```
+	dfx start
+	```
+4. `index.html` and `index.js` is also generated by `dfx new my_dapp` command. These files can be found inside the `src/my_dapp_frontend/src/`.
+
+5. Install the dependencies included in this generated project files by running.
+	```
+	npm install

```

Comment:
Bakit nag `npm install` pa, by default may `node_modules` na upon initially generating the project via `dfx new`? Wala din pati binago sa `package.json`.